pkg/ccl/serverccl/admin_test.go

```diff
@@ -93,3 +108,48 @@ func TestAdminAPIChartCatalog(t *testing.T) {
 	err := serverutils.GetJSONProto(firstServer, adminPrefix+"chartcatalog", &resp)
 	require.NoError(t, err)
 }
+
+func TestAdminAPIJobs(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+
+	//ctx := context.Background()
+	dir, dirCleanupFn := testutils.TempDir(t)
+	defer dirCleanupFn()
+	s, conn, _ := serverutils.StartServer(t, base.TestServerArgs{ExternalIODir: dir})
+	defer s.Stopper().Stop(context.Background())
+	sqlDB := sqlutils.MakeSQLRunner(conn)
+
+	sqlDB.Exec(t, `BACKUP INTO 'nodelocal://0/backup/1?AWS_SECRET_ACCESS_KEY=neverappears'`)
+
+	var jobsRes serverpb.JobsResponse
+	if err := getAdminJSONProto(s, "jobs", &jobsRes); err != nil {
+		t.Errorf(err.Error())
+	}
+
+	var backups []serverpb.JobResponse
+	for _, job := range jobsRes.Jobs {
+		if job.Type != "BACKUP" {
+			continue
+		}
+		backups = append(backups, job)
+
+	}
+
+	if len(backups) != 1 {
+		t.Errorf("Expected 1 Backup job, got %d", len(backups))
+	}
+
+	jobID := backups[0].ID
+
+	var jobRes serverpb.JobResponse
+	path := fmt.Sprintf("jobs/%v", jobID)
+	var err error
+	if err = getAdminJSONProto(s, path, &jobRes); err != nil {
```

Comment:
No need to pre-declare err here:

```suggestion
	if err := getAdminJSONProto(s, path, &jobRes); err != nil {
```